### PR TITLE
Task 69 multiple listings backend

### PR DESCRIPTION
**Pull request by Rebooting-Me**
This PR completes the backend tasks for Sprint 2 with CO able to view multiple listings and CO able to filter out listings accordingly.

### CODE REVIEW

**Comment by Rebooting-Me** [comment]
This PR closes these above tasks.

File: backend/routes/contractor.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ router.get('/profile/:contractor_id', getContractorProfile);
 router.use(authorizeContractor);
 
 // routes
+
+// Allows CO to view all listings in database (Filter feature included)
+router.post("/listings", viewMultipleListings );
```
Comment (and-le) [inline backend/routes/contractor.js:26]: Should this be POST or get? I see that we send the filters we want to use, but we aren't actually changing any data on the server side. Would GET be more apt?

File: backend/.env [hunk 2]
```diff
@@ -1 +1,2 @@
-SECRET="secret_string"
\ No newline at end of file
+SECRET="secret_string"
+MONGO_URI = "mongodb://localhost:27017/"
```
Comment (and-le) [inline backend/.env:2]: The URI is not needed in the .env file (when committing); please remove it. It is OK to have it locally.

File: backend/server.js [hunk 3]
```diff
@@ -6,12 +6,12 @@ const app = require('./app');
 require('dotenv').config();
 
 // Database
-const databaseUri = process.env.MONGODB_URI;
-mongoose.connect(databaseUri, { useNewUrlParser: true })
+const db = process.env.MONGO_URI;
```
Comment (and-le) [inline backend/server.js:9]: Please use the existing lines for defining the URI:

`const databaseUri = process.env.MONGODB_URI;
mongoose.connect(databaseUri, { useNewUrlParser: true })`

**Review by and-le — CHANGES_REQUESTED** [review]
Nice work on the listings! Please see the comments.

File: backend/routes/contractor.js (hunk 1 above)
Comment (Rebooting-Me) [reply backend/routes/contractor.js:26]: Yes. This should be POST as GET method should not have a request body according to HTTP specifications. And we want a request body for specifying the filters.

File: backend/.env (hunk 2 above)
Comment (Rebooting-Me) [reply backend/.env:2]: Oh got it.

File: backend/server.js (hunk 3 above)
Comment (Rebooting-Me) [reply backend/server.js]: Done

**Review by and-le — APPROVED** [review]
Looks good!

**Review by and-le — APPROVED** [review]

**Pull request merged**